Source/Core/Core/BootManager.cpp

```diff
@@ -300,6 +305,9 @@ bool BootCore(const std::string& _rFilename)
         g_wiimote_sources[WIIMOTE_BALANCE_BOARD] = source;
         WiimoteReal::ChangeWiimoteSource(WIIMOTE_BALANCE_BOARD, source);
       }
+
+      IniFile::Section* sysconf_section = game_ini.GetOrCreateSection("Wii");
```

Comment:
Sorry, I missed this before, but the variable name should be `wii_section` (since the section isn't called [Sysconf] anymore)